docker/docker-compose.yaml

```diff
@@ -162,6 +163,26 @@ services:
     networks:
       - von
 
+  cypress:
+    image: "cypress/included:3.2.0"
+    depends_on:
+      - react-holder
+      - react-issuer
+      - react-verifier
+      - react-authority
+    environment:
+      - CYPRESS_TEST_USERNAME=alice@drone-pilot.com
+      - CYPRESS_TEST_PASSWORD=dronePilot01
```

Comment:
These probably shouldn't be committed

Reply:
updated account details as well, thanks for highlighting